rpc/rpcserver/server.go

```diff
@@ -623,6 +756,27 @@ func marshalHashes(v []*chainhash.Hash) [][]byte {
 	return hashes
 }
 
+func marshalHashStringList(v []string) ([][]byte, error) {
+	// Check the size of the inputs.
+	for i := range v {
+		if len(v[i]) != (chainhash.HashSize * 2) {
+			return nil, errors.New("bad size of hash string item")
```

Comment:
include the item, the index, or the length, or all of the above.

Better yet, return actual hashes so this nonsense isn't necessary, and instead you just create slices that will always be the right size.